inspect.cpp

```diff
@@ -665,6 +665,17 @@ namespace Sass {
     append_token("null", n);
   }
 
+  void Inspect::operator()(Parent_Selector* p)
+  {
+    if (p->selector()) {
+      p->selector()->perform(this);
+      append_string(";");
+    }
+    else {
+      append_string("&");
```

Comment:
If there should be an optional space, be sure to call `append_optional_space()` too!